src/schema/sources.ts

```diff
@@ -344,6 +354,21 @@ export const typeDefs = /* GraphQL */ `
       role: String!
     ): EmptyResponse! @auth
 
+    """
+    Unblock a removed member with blocked role
+    """
+    unblockMember(
```

Comment:
Since this simply removes a member from the squad, can we not leverage the existing "leaveSquad" function?

Reply:
This makes sense, though leave refers to the logged-in user. Would love to hear more thoughts on this if can just make it accept a parameter. cc @capJavert @idoshamun

Reply:
I am in favor of dedicated function because down the line what unblocking does could change or add more things.

Logically like this, leaving squad is just one part of the unblocking flow, which makes more sense to me.

Reply:
Alright, let's leave it for now.
How do you feel naming wise? `unblockMember` is not actually the right term in my opinion.
It totally removes someone from a squad as member.

Reply:
Hmm. I think the intent with `unblock` is clear. But I'd love to hear suggestions on what we can replace it with.

Reply:
I'm just not a fan of the product decision to call it unblock, as it's really not what it does.
To me the whole action is "remove member from squad", but let's leave it, it's nitpicking at this point.